docusaurus/docs/how-to-guides/app-plugins/customize-navigation-position.md

```diff
@@ -0,0 +1,90 @@
+---
+id: customize-navigation-position
+title: Customize navigation placement of plugin pages
+description: How to relocate Grafana app plugin pages to customize the navigation menu structure.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - navigation
+  - customize
+  - configuration
+  - grafana.ini
+---
+
+# Customize Navigation Placement of App Plugin Pages
+
+# Introduction
+
+By default, Grafana app plugins and their pages appear under the "More apps" section in the navigation menu. However, as a Grafana administrator, you might want to improve user experience by relocating specific pages or entire app plugins to more relevant sections of the navigation hierarchy. This guide shows you how to customize the placement of app plugin pages across different parts of your Grafana navigation menu.
+
+:::important
+The navigation customization described in this guide must be performed by the Grafana administrator in the Grafana instance configuration. Plugin developers cannot directly control the navigation placement in a Grafana instance.
```

Comment:
Hmm... if this is out of the developers control is this developer or user related documentation? This doc feels much more rounded with examples than the section that exists in our user docs [configure-grafana#navigationapp_sections](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#navigationapp_sections). Maybe should move it there?